flow/surface_frame.cc

```diff
@@ -31,9 +32,13 @@ SurfaceFrame::SurfaceFrame(sk_sp<SkSurface> surface,
     canvas_ = &adapter_;
   } else if (display_list_fallback) {
     FML_DCHECK(!frame_size.isEmpty());
-    dl_builder_ =
-        sk_make_sp<DisplayListBuilder>(SkRect::Make(frame_size), true);
-    canvas_ = dl_builder_.get();
+#if IMPELLER_SUPPORTS_RENDERING
+    aiks_canvas_ =
+        std::make_shared<impeller::DlAiksCanvas>(SkRect::Make(frame_size));
+    canvas_ = aiks_canvas_.get();
+#else
+    FML_DCHECK(false);
```

Comment:
The flag is kind of misnamed now. It really means "we're rendering in Impeller mode, so do the Impeller thing". It was named after the (former) solution rather than the trigger that caused it...